client/src/modules/stock/lots/templates/row.expired.html

```diff
@@ -4,7 +4,12 @@
   ng-repeat="(colRenderIndex, col) in colContainer.renderedColumns track by col.uid"
   ui-grid-one-bind-id-grid="rowRenderIndex + '-' + col.uid + '-cell'"
   class="ui-grid-cell"
-  ng-class="{ 'ui-grid-row-header-cell': col.isRowHeader, 'expired': row.entity.lifetime < 0 }"
+  ng-class="{ 'ui-grid-row-header-cell': col.isRowHeader, 
+              'at-risk-of-expiring': row.entity.quantity > 0 && row.entity.lifetime >= 0 && row.entity.IS_IN_RISK_EXPIRATION,
```

Comment:
Thanks for the suggestions.  I had wanted to use a function but had not quite figured it out this way.   When I figured out this short cut in the template I just went with it..    I'm glad to do it the way you suggest.

Reply:
I implemented these changes.  I did go with the same logic I had before, so that will probably need to be addressed in the future.   In the lots registry, we may want display then with their parent inventory items by grouping lots under their respective inventory item and applying some highlighting to the parent inventory item and some to the child lots, as appropriate.